apps/api/src/api/art/controllers/custom.ts

```diff
@@ -66,6 +60,27 @@ export default {
   async unlike(ctx: Context) {
     await checkRecaptcha(ctx)
 
+    const profile = await getProfile(ctx)
+
+    if (profile) {
+      const isLikedCount = await strapi.entityService.count('api::art.art', {
+        filters: {
+          id: { $eq: ctx.params.id },
+          likers: { id: { $in: [profile.id] } },
```

Comment:
It seems there was some misunderstanding about the use of the `$in` operator here. Ideally, we can use the `$in` operator to check if multiple provided ids are in the `likers` relation. However, we will always provide a single profile id. If `$eq` works as well, I'd use it. It could also be better for performance.

```suggestion
          likers: { id: { $eq: profile.id } },
```